include/aarch64/gpio.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _GPIO_H_
```

Comment:
Should you add prefix `bcm2835_` for all the definitions below ?

Reply:
Done.